service/bin/prepare

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# The goal of this script is to prepare
+
+set -xe
+
+# ----
+# NGINX
+# As nginx is only supplementary to the workload of the charm.
+# As such, its files are handled in SNAP_DATA
+
+# Copy nginx configuration
+cp -r --preserve=mode "$SNAP/nginx" "$SNAP_DATA/nginx"
+
+# Create some directories for nginx
+mkdir -p $SNAP_DATA/nginx/tmp/body
```

Comment:
Can you put quotes around these paths? It's considered good practice

Reply:
This is coming in a subsequent PR: https://github.com/canonical/charmed-bind/pull/23